### DISCUSSION TITLE
Bad Session Description on FreeSWITCH TCP

### DISCUSSION THREAD

**Discussion by KORuL** [baresip/baresip#2664] [Q&A]
I'm trying to make a TCP call via FreeSwitch + Android application using baresip. Registration is successful, then sdp is sent, but FreeSwitch doesn't like something

Please Help

INVITE sip:10893611@37.139.43.14:8060;transport=tcp SIP/2.0
Via: SIP/2.0/TCP 10.215.173.1:41744;branch=z9hG4bK29d37cc4d3f65bfe;rport
Contact: <sip:55942081-0x7b1d365690@10.215.173.1:41744;transport=tcp>
Max-Forwards: 70
To: <sip:10893611@37.139.43.14:8060;transport=tcp>
From: <sip:55942081@37.139.43.14:8060>;tag=323bbc8185b6bd04
Call-ID: e745e86d0fa8e452
CSeq: 43192 INVITE
User-Agent: uki
Allow: INVITE,ACK,BYE,CANCEL,OPTIONS,NOTIFY,INFO,MESSAGE,UPDATE
Supported: gruu,replaces
Content-Type: application/sdp
Content-Length: 687

o=- 4283199382 1548184820 IN IP4 10.215.173.1
c=IN IP4 10.215.173.1
m=audio 62933 RTP/AVP 8 0 101
c=IN IP4 82.146.52.7
a=rtpmap:8 PCMA/8000
a=rtpmap:0 PCMU/8000
a=rtpmap:101 telephone-event/8000
a=fmtp:101 0-15
a=rtcp:61697 IN IP4 82.146.52.7
a=sendrecv
a=rtcp-rsize
a=ssrc:1586807132 cname:sip:55942081@37.139.43.14:8060
a=mid:0
a=minptime:20
a=ptime:20
a=label:1
m=video 59304 RTP/AVP 96
c=IN IP4 149.154.66.115
a=rtpmap:96 H264/90000
a=fmtp:96 packetization-mode=0;profile-level-id=42e01f
a=rtcp:63076 IN IP4 82.146.52.7
a=recvonly
a=rtcp-rsize
a=ssrc:3943336433 cname:sip:55942081@37.139.43.14:8060
a=mid:1
a=rtcp-fb:* nack
a=rtcp-fb:* nack pli
a=label:2

SIP/2.0 400 Bad Session Description
Via: SIP/2.0/TCP 10.215.173.1:41744;branch=z9hG4bK29d37cc4d3f65bfe;rport=55782;received=213.87.151.46
From: <sip:55942081@37.139.43.14:8060>;tag=323bbc8185b6bd04
To: <sip:10893611@37.139.43.14:8060;transport=tcp>;tag=Bmy65UtrKXe3K
Call-ID: e745e86d0fa8e452
CSeq: 43192 INVITE
Accept: application/sdp
Allow: INVITE, ACK, BYE, CANCEL, OPTIONS, MESSAGE, INFO, UPDATE, REGISTER, REFER, NOTIFY, PUBLISH, SUBSCRIBE
Supported: timer, path, replaces
Allow-Events: talk, hold, conference, presence, as-feature-event, dialog, line-seize, call-info, sla, include-session-description, presence.winfo, message-summary, refer
Content-Length: 0

**Comment by KORuL** [comment]
![2023-07-28 17 38 18](https://github.com/baresip/baresip/assets/19871487/9602c4e1-3b42-4ab9-b106-501fdc714361)
Did I select correctly? maybe because of this

**Comment by alfredh** [comment]
please follow the guidelines here:

https://github.com/baresip/baresip/wiki/Guidelines-for-reporting-Issues


Is this happening with stock baresip client ?
